mainwindow.cc

```diff
@@ -1226,13 +1225,21 @@ void MainWindow::applyQtStyleSheet( QString const & displayStyle, QString const
   css += macCssFile.readAll();
 #endif
 
-  if ( displayStyle.size() )
-  {
-    // Load an additional stylesheet
-    QFile builtInCssFile( QString( ":/qt-style-st-%1.css" ).arg( displayStyle ) );
-    if ( builtInCssFile.open( QFile::ReadOnly ) )
-      css += builtInCssFile.readAll();
-  }
+#if defined(Q_OS_WIN)
```

Comment:
I think ,this section can put into a seperator qt css file. such as qt-style-windows.
and load this file when in Window platform

Reply:
Sure. I also keep the `qt-style-mac.css` there in case we need adjustments later.

Reply:
I forced push the change